.travis.yml

```diff
@@ -19,11 +19,6 @@ matrix:
 #      env:
 #        - PYTHON_VERSION=3.4.5
```

Comment:
There are however conda packages for SimpleITK, would it make sense to get the mac tests working on miniconda/anaconda (https://anaconda.org/simpleitk/simpleitk) instead of standard python?